src/main/java/uk/gov/cslearning/record/service/NotifyService.java

```diff
@@ -64,7 +64,7 @@ public void notifyOnComplete(String email, String templateId, String learner, St
 
             LOGGER.debug("Complete notify email sent: {}", response.getBody());
         } catch (NotificationClientException e) {
-            LOGGER.error("Could not send email to GOV notify: ", e.getLocalizedMessage());
+            LOGGER.error("Could not send email to GOV notify: {}", e.getLocalizedMessage());
```

Comment:
Should we log localized message? It could contain sensitive data regarding notification client.

Reply:
Currently we don't get that detail and that caused us to have to ask Notify for the server-side issue so this would definitely be needed for error handling.

Reply:
Ok